### PR TITLE
{math}[GCCcore/11.3.0,foss/2022a] fpylll v0.5.9, fplll v5.4.5

### PR DESCRIPTION
**Pull request by migueldiascosta**
(created using `eb --new-pr`)

- [x] requires #19349

### CODE REVIEW

**Comment by migueldiascosta** [comment]
@boegelbot please test @ generoso

**Comment by migueldiascosta** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/f/fplll/fplll-5.4.5-GCCcore-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+easyblock = 'ConfigureMake'
+
+name = 'fplll'
+version = '5.4.5'
+
+homepage = 'https://github.com/fplll/fplll'
+description = """fplll contains implementations of several lattice algorithms.
+ The implementation relies on floating-point orthogonalization,  and the 1982 paper from 
+Lenstra, Lenstra Jr. and Lovasz (LLL( is central to the code, hence the name."""
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/f/fplll/fplll-5.4.5-GCCcore-11.3.0.eb:9]:
```suggestion
Lenstra, Lenstra Jr. and Lovasz (LLL) is central to the code, hence the name."""
```

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
jsczen3c1.int.jsc-zen3.fz-juelich.de - Linux Rocky Linux 9.3, x86_64, AMD EPYC-Milan Processor (zen3), Python 3.9.18
See https://gist.github.com/SebastianAchilles/94be4d4fdb155a44ed89559c9f078cf1 for a full test report.

**Review by SebastianAchilles — APPROVED** [review]
lgtm

**Comment by SebastianAchilles** [comment]
Going in, thanks @migueldiascosta!

**Pull request merged**